flow/probes/sflow.go

```diff
@@ -126,6 +130,9 @@ func (probe *SFlowProbe) Start() error {
 	// start index/mac cache updater
 	go probe.cacheUpdater()
 
+	flowtable := flow.NewFlowTable()
+	go flowtable.AsyncExpire(probe.flowExpire, 5*time.Minute)
```

Comment:
They will expire, even with an empty callback
See my comment (on the server side) above